debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleSchemaChangeEventEmitter.java

```diff
@@ -105,6 +109,14 @@ public void emitSchemaChangeEvent(Receiver receiver) throws InterruptedException
                         case DROP_TABLE:
                             changeEvents.add(dropTableEvent(partition, tableBefore, (TableDroppedEvent) event));
                             break;
+                        case TRUNCATE_TABLE:
+                            if (truncateReceiver != null) {
+                                truncateReceiver.processTruncateEvent();
+                            }
+                            else {
+                                LOGGER.warn("Truncate event without truncateReceiver set. Ignoring.");
+                            }
```

Comment:
from the point of the code: you could get in here if you get a truncate event during the snapshotting. But I don't think that's possible? Since I was not 100% sure, I made it a WARN.

Reply:
Gotcha; so no, we'd never receive a truncate event during snapshotting, and we can remove this safe-guard (the reason why I'm in favour of it is that these sorts of things tend to confusion down the road; having that safe-guard in place suggests that there actually _could_ be a situation where that's the case).

Reply:
ok removed it